apps/console/src/features/organizations/pages/organization-roles-edit.tsx

```diff
@@ -0,0 +1,85 @@
+import { PageLayout } from "@wso2is/react-components";
```

Comment:
Add license header

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/3617/commits/81488c55dd6c9c7a63575dee78e71f2b80643963